frontend/src/services/auth/sessionUtils.ts

```diff
@@ -0,0 +1,46 @@
+import { KeyObject } from "crypto";
+import { JWTPayload, jwtVerify, SignJWT } from "jose";
+
+import { cookies } from "next/headers";
+
+export const CLIENT_JWT_ENCRYPTION_ALGORITHM = "HS256";
+export const API_JWT_ENCRYPTION_ALGORITHM = "RS256";
```

Comment:
Should these live in `src/constants`? If not what is our criteria for what goes there vs not?